### DISCUSSION TITLE
Role terminology confusion when creating a Node from MetalK8s UI

### DISCUSSION THREAD

**Discussion by thomasdanan** [scality/metalk8s#3170] [General]
When looking at a Node from CLI (`kubectl get node <name>`) or from the UI we can get the [**Node roles**](https://metal-k8s.readthedocs.io/en/latest/installation/introduction.html#node-roles):

<img width="347" alt="Screenshot 2021-03-04 at 18 34 00" src="https://user-images.githubusercontent.com/29035069/110005285-99ec0100-7d18-11eb-92e6-b9af671b80e2.png">

When we create a Node from MetalK8s UI, the user is asked to choose between 3 **roles**:

<img width="385" alt="Screenshot 2021-03-04 at 18 33 48" src="https://user-images.githubusercontent.com/29035069/110004951-44afef80-7d18-11eb-925b-1829f386abb3.png">

However, those roles are not the node roles and it confuses the user:

```
Roles/Workload Plane: enable any workload applications run on this Node
Roles/Control Plane: enable master and etcd services run on this Node
Roles/Infra: enable infra services run on this Node
```

Don't you think it would make more sense to use a different label or directly use the description above?

**Comment by NicolasT** [comment]
Internally we have the 4 distinct roles (and according labels, taints, what gets deployed/bound where,...), and we should stick to those. We decided not to expose all 4 of them in the UI and instead bundle `etcd` and `master` up into _Control Plane_ to simplify things for a UI user: in general you'll want to have `etcd` and `master` (which we should likely rename as well) colocated, but we still have the option not to.
